src/python/deepgnn/graph_engine/snark/local.py

```diff
@@ -118,8 +122,16 @@ def sample_neighbors(
                 default_weight,
                 seed=int(random.getrandbits(64)),
                 timestamps=timestamps,
+                return_edge_created_ts=return_edge_created_ts,
+            )
+            if return_edge_created_ts:
```

Comment:
Should be if not return_edge_created_ts?

Reply:
No. If `return_edge_created_ts` is set to `True` we want to return everything, because tuple will have 4 elements. Otherwise, the tuple will have only 3 elements and we need to backfill.